### ISSUE TITLE
Option for removing line breaks ?

### ISSUE THREAD

**Issue by vvo** [kangax/html-minifier#22]
Do you think it would be interesting to remove all line breaks ?

It should take care of 
    <pre> tags.

**Comment by kangax** [comment]
Remove line breaks where exactly? They are already removed in places where they are insignificant. For example, `<p>\nfoo</p>` is minified to `<p>foo</p>` and `<p class="\n\nfoo \nbar">` is minified to `<p class="foo bar">`

**Comment by vvo** [comment]
Ah well the "collapse whitespace" option is doing it but also removes space before an a tag in a p

```
<p>test <a>s</a></p> becomes <p>test<a>dsqd</a></p> (bad)
```

I saw you have blogged on this problem, sorry. I can't do like you say in the blog post, adding margins everywhere!

I had deactivated the collapseWhitespace option and thought a simple remove line breaks would be sufficient to get rid of a lot of bytes.

```
<p>test</p>

<p>test2</p>
```

becomes

```
<p>test</p><p>test2</p>
```

I'll need to think more before creating an issue.
